config/app.cfg

```diff
@@ -22,3 +22,6 @@ webhook_key = some_string
 region = us-east-1
 subscriber = 512686554592
 name = bus-profiles
+env = dev
+# TODO: not implemented yet?
+endpoint = http://goaws:4100
```

Comment:
@seanwiseman you were testing locally with `goaws`, so how do we extend `bus-sdk-python` to accept the `endpoint[ url]` for local usage?

Reply:
All you need to do is pass in `endpoint_url` as a config override and the initialization is handled for you.
```
from elife_bus_sdk import get_publisher

config = {
    'region': 'us-east-2',
    'subscriber': '00000000000',       
    'name': 'profile',
    'endpoint_url': 'http://goaws:4100'  # or from a config variable etc
}

publisher = get_publisher(pub_type='sns', config=config)
```
This would give you an SNS 'publisher' pointed at your local `goaws` instance. This will be the same with SNS queues.